requirements.txt

```diff
@@ -1,3 +1,4 @@
 pymavlink>=2.4.14
 pyserial>=3.0
 numpy
+openai
```

Comment:
instead of in requirements I'd prefer it give a message on load "need to install openai with 'python -m pip install openai'

Reply:
ok, removed openai from requirements.